config.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Configuration
+layout: sidebar
+category: intro
+---
+
+## Configuration
+
+Testium uses [`rc`](https://www.npmjs.com/package/rc) for loading configuration.
+Be aware that command line arguments won't work
+since most test runners (e.g. `mocha`) have their own argv handling.
+
+### Top-Level Options
+
+#### `browser`
+
+The name of the browser to use.
+Expects a valid webdriver `browserName`.
+If no browser is specified, testium will assume it's `'phantomjs'`.
+
+#### `desiredCapabilities`
+
```

Comment:
"Additional desired webdriver capabilities" doesn't explain what this does. It's like writing a dictionary definition of a word by restating the word.

What does this option actually do? Why would I want to use it?